content/de/pressreview/2018-06-05_zurichsee-zeitung.md

```diff
@@ -3,4 +3,4 @@ Slug: 2018-06-05_zurichsee-zeitung
 Date: 2018-06-05
 Lang: de
 Publisher: Zürichsee-Zeitung
-Ext_url: https://www.zsz.ch/obersee/EVoting-nimmt-erste-Huerde/story/25589562
\ No newline at end of file
+Ext_url: https://files.coredump.ch/medien/20180605_zsz.pdf
```

Comment:
Do we prefer online articles (paywalled except with uBlock Origin) or PDFs?

Reply:
for archival purposes I'd say PDF - but that's not really the difference you're after